### ISSUE TITLE
Create basic pages and routes for user dashboard pages

### ISSUE THREAD

**Issue by Eakam1007** [Seneca-CDOT/starchart#138]
As @humphd mentioned in one of our meetings, it would be nice to have the routes setup for different pages so that the UI design could be implemented easily in parallel. The pages served from these routes do not need to have anything more than a simple heading stating what the page is.

This issue focuses on adding the following routes (based on #36):

- /domains - view and manage domains
- /domains/:id - edit subdomain
- /domains/new - create new subdomain
- /certificate - view, renew, or request a certificate (A user can only have one certificate)
- /certificate/instructions - help page about certificates (see #114)

**Comment by humphd** [comment]
This is great!  I'm happy to see this happening, and we should get it landed sooner rather than later in the 0.2 cycle.

**Comment by Eakam1007** [comment]
@humphd Should I also add authorization for these routes, or leave them to be basic skeleton pages?

**Comment by humphd** [comment]
@Eakam1007 leave it, @sfrunza13 can do it as part of the other work.
